src/Server.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Server.hpp"
+#include "nlohmann/json.hpp"
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include "curl/curl.h"
+#include "Utility.hpp"
+#include "User.hpp"
+
+std::string data;
+
+size_t write_data(char* buf, size_t size, size_t nmemb, void* up)
+{
+    for (size_t c = 0; c < size*nmemb; c++)
+        data.push_back(buf[c]);
+    return size * nmemb;
+}
+
+void Server::downloadUsers()
+{
+    CURL* curl;
```

Comment:
Can unique_ptr be used here?

Reply:
It would be great, but i think that CURL library is not prepared for unique pointers in their functions. I've got a lot of errors when i push unique_ptr inside. Testing is still on way.